src/integrations/stripe.ts

```diff
@@ -15,82 +17,91 @@ const stripePromise = new Promise((resolve) => {
 export const getStripe = () => stripePromise;
 
 export function installStripe(stripeConfig: any) {
-  // eslint-disable-next-line camelcase
-  if (stripeConfig?.metadata?.client_key) {
-    document.dispatchEvent(
-      new CustomEvent('stripe_key_loaded', {
-        detail: { key: stripeConfig.metadata.client_key }
-      })
-    );
+  const metadata = stripeConfig?.metadata;
+  if (metadata) {
+    // eslint-disable-next-line camelcase
+    if (metadata.client_key) {
+      document.dispatchEvent(
+        new CustomEvent('stripe_key_loaded', {
+          detail: { key: stripeConfig.metadata.client_key }
+        })
+      );
+    }
+    // If a total hidden field is configured, then initialize it to any addition cost
```

Comment:
this seems like it could be done on the backend instead

Reply:
At the beginning, the hidden field total does not have a value yet (until a product is selected and total calculated). Should I find stripe integration, check the total, create a hiddenfieldvalue for it here? https://github.com/feathery-org/feathery-backend/blob/master/apps/api/views.py#L280

Reply:
Moved to BE.